rundeckapp/grails-app/services/rundeck/services/ExecutionService.groovy

```diff
@@ -3804,17 +3812,14 @@ class ExecutionService implements ApplicationContextAware, StepExecutor, NodeSte
                                 execution : execution,
                                 nodestatus: [succeeded: sucCount, failed: failedCount, total: newContext.getNodes().getNodeNames().size()],
                                 context   : newContext,
-                                jobref    : jitem.jobIdentifier
+                                jobref    : jitem.jobIdentifier,
+                                export    : data
                         ]
                 )
             }
 
             result.sourceResult = wresult.result
 
-            Map<String, String> data = ((WorkflowExecutionResult) wresult.result)?.getSharedContext()?.getData(ContextView.global())?.get("export")
-            if (data) {
-                executionContext.getOutputContext().addOutput(ContextView.global(), "export", data)
-            }
```

Comment:
i think this will break exported data vars for job references in the case where `jitem.ignoreNotifications` is true